docs/tutorials/creating_a_test_environment.md

```diff
@@ -1,4 +1,7 @@
-Hosts/Nodes/SUTs are defined in the --hosts (--config) file in Yaml format.  This file defines each node in the test configuration. The file can be saved anywhere and used with `beaker --hosts yourhost.cfg` see [The Command Line](The-Command-Line.md) for more info.
+Hosts/Nodes/SUTs are defined in the --hosts (--config) file in Yaml format.
+This file defines each node in the test configuration. The file can be saved
+anywhere and used with `beaker --hosts yourhost.cfg`
```

Comment:
let's be uniform about the filename extension we use for host configs.  beaker outputs them as .yaml (and in certain cases .yml :-\).  perhaps this should say `yourhost.yaml`?